src/main.ts

```diff
@@ -22,19 +21,36 @@ import { processSources } from './processor';
     });
 })().catch(console.error);
 
-function getArguments() {
+function getArguments(): Config {
     return yargs(hideBin(process.argv))
+        .strict()
         .options({
-            'base': {
+            base: {
                 type: 'string',
-                description: 'Directory of your root',
+                description: 'The target root directory',
                 default: './',
             },
-            'contracts': {
+            contracts: {
                 type: 'string',
-                description: 'Directory of your solidity contracts',
+                description: 'The directory containing your Solidity contracts',
                 required: true,
             },
+            enforceInheritdoc: {
+                type: 'boolean',
+                description: 'When set to true, all interface methods must have natspec',
+                default: true,
+            },
+            constructorNatspec: {
+                type: 'boolean',
+                description: 'True if constructor natspec is mandatory',
+                default: true,
+            },
+            ignore: {
```

Comment:
are there people who have their tests in the same directory as their core smart contracts? If there are, this functionality would not work

Reply:
what about using glob instead?